src/sections/offering/page-down.tsx

```diff
@@ -0,0 +1,23 @@
+import { Wrapper } from 'components/offering/wrapper'
```

Comment:
Can we maybe rename this file (`page-down`) to `PageDown`? A) i think we should be consequent with camel case/pascal case or kebab case, and personally at least i prefer camel/pascal. And since this only exports one component i think the name should be the same as the component

Reply:
Good idea - PascalCase is probably preferable I think as well